packages/flutter_tools/lib/src/vscode/vscode_validator.dart

```diff
@@ -14,6 +14,7 @@ class VsCodeValidator extends DoctorValidator {
   VsCodeValidator(this._vsCode) : super(_vsCode.productName);
 
   final VsCode _vsCode;
+  final UserMessages _userMessages = UserMessages();
```

Comment:
should this not be a global or direct injected?

Reply:
I prefer this as local because it makes it obvious that the messages being retrieved are not context-dependent.

Reply:
But won't this lead to someone overriding these messages THINKING they've overridden them when they have not?

Reply:
Proposal: move these messages from `UserMessages` to this file

Reply:
perfect